### PR TITLE
Receiver: report back final revision after commit

### PR DESCRIPTION
**Pull request by janpriessnitz**

### CODE REVIEW

**Comment by jblomer** [comment]
@cernvm-bot cloudtest

File: test/unittests/t_catalog_merge_tool.cc [hunk 1]
```diff
@@ -121,7 +121,8 @@ TEST_F(T_CatalogMergeTool, CRUD) {
   EXPECT_TRUE(merge_tool.Init());
 
   std::string output_manifest_path;
-  EXPECT_TRUE(merge_tool.Run(params, &output_manifest_path));
+  uint64_t final_rev;
+  EXPECT_TRUE(merge_tool.Run(params, &output_manifest_path, &final_rev));
```
Comment (jblomer) [inline test/unittests/t_catalog_merge_tool.cc:125]: We should check for the correct `final_rev` in this test.

**Comment by janpriessnitz** [comment]
@cernvm-bot cloudtest

**Comment by jblomer** [comment]
@cernvm-bot cloudtest

**Pull request merged**